Tasks/AzureFileCopyV5/README.md

```diff
@@ -0,0 +1,118 @@
+
+  * **Copy in Parallel**: Selecting this option will copy files to all the VMs in the Resource Group in-parallel, hence speeding up the process of copying. 
+  
+  * **Clean Target**: Selecting this option will clean the destination folder prior to copying the files to it. When destination type is AzureBlob, it will delete all the folders/files in that container and If blob prefix is provided then it will delete all the folders/files in that blob prefix.
+
+  * **Test Certificate**: This setting is required while copying the files from the blob containers to the Azure VMs. The copy operation is initiated over the WinRM HTTPS protocol and if the VM has a test certificate installed on it, then select this option to  to skip the validation that the server certificate is signed by a trusted certificate authority (CA).  
+
+* **Destination**: If the target is Azure blob then the following parameters need to be filled out. 
+
+  * **Container Name**: The name of the container where the files will be copied to. If the container does not exist then a new one will be created with the name provided in this parameter. 
+
+  * **Blob Prefix**: A prefix for the Blobs that can be used to filter the blobs like appending the Build number to the blobs, so that all the blobs with the same build number can be downloaded from the Container.
+
+* **Additional Arguments**: Additional [AzCopy.exe](https://azure.microsoft.com/en-us/documentation/articles/storage-use-azcopy/) arguments that will be applied for uploading to blob and same will be applied for downloading while copy to VM.
+  * **Blob Destination**: Supported additional arguments for copy to blob are /BlobType:, /Pattern:, /L, /Z, /XN, /A, /IA:, /XA:, /NC:, /DestType: and /SetContentType.
+ 
+  * **VM Destination**: Supported additional parameters for copy to VM are /Pattern:, /L, /NC: and /XN.
+ 
+### Output Variables
+
+The task creates the following as output variables:
+* **StorageContainerUri**: When copying files to an Azure container, this parameter returns the Uri of the container were the files were copied to.
+* **StorageContainerSasToken**: When copying files to an Azure container, a SasToken is created and returned.  By default, this token expires after 4 hours.
+
+Following the [output variables](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#use-output-variables-from-tasks) documentation and naming the step, we can then reference the variables like so:
+
+```yaml
+- task: AzureFileCopy@5
+  inputs:
+    SourcePath: 'Readme.md'
+    azureSubscription: 'Azure'
+    Destination: 'AzureBlob'
+    storage: 'storageAccount'
```

Comment:
provide sample yaml for mimetype

Reply:
Same YAML can be used for mime type. no additional changes required for mime type.